components/public-api-server/pkg/apiv1/oidc_test.go

```diff
@@ -43,19 +43,16 @@ var (
 		},
 	}
 
-	user = newUser(&protocol.User{})
+	user           = newUser(&protocol.User{})
+	organizationID = uuid.New()
```

Comment:
We re-use these two across the tests for the default scenario - with user being org owner. There are are extra tests which test the negative case